### PR TITLE
Implement You Might Like/Suggestions

### PR DESCRIPTION
**Pull request by psoleckimoj**
Present 4 items matching from secondary tags then primary tags on content pages

- Add suggested content repo/service
- Add call to suggestions
- Add component for suggested content

### CODE REVIEW

File: moj-node/server/services/hubContent.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ module.exports = function createHubContentService({
     }
 
     const content = await contentRepository.contentFor(id);
+
+    const suggestedContent = await contentRepository.suggestedContentFor({
```
Comment (romidane) [inline moj-node/server/services/hubContent.js]: Don't think you need this call here?

File: moj-node/test/assets/renderSuggestedItem.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const { createFeaturedItem } = require('../test-helpers');
```
Comment (romidane) [inline moj-node/test/assets/renderSuggestedItem.spec.js]: I don't think this is necessary

File: moj-node/test/assets/templates/suggestedContent.tmpl.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+module.exports = `
```
Comment (romidane) [inline moj-node/test/assets/templates/suggestedContent.tmpl.js]: I think you can get rid of this?

File: moj-node/server/services/hubContent.js (hunk 1 above)
Comment (romidane) [reply moj-node/server/services/hubContent.js]: since you make the same call in the media endpoint

Comment (psoleckimoj) [reply moj-node/server/services/hubContent.js]: Being passed in to a default content item as the ticket said all content items

Comment (psoleckimoj) [reply moj-node/server/services/hubContent.js]: Ahh yes, I'll change 👍

Comment (romidane) [reply moj-node/server/services/hubContent.js]: right so the remove the extra call in the media fn?

File: moj-node/server/services/hubContent.js [hunk 4]
```diff
@@ -60,7 +74,7 @@ module.exports = function createHubContentService({
     };
   }
 
-  async function landingPage(data, establishmentId) {
+  async function landingPage(establishmentId, data) {
```
Comment (romidane) [inline moj-node/server/services/hubContent.js]: nice

Comment (psoleckimoj) [reply moj-node/server/services/hubContent.js]: It annoyed me 😆

File: moj-node/server/views/components/suggested-content/suggested-content.njk [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{% from "suggested-content/macro.njk" import hubSuggestedContent %}
```
Comment (romidane) [inline moj-node/server/views/components/suggested-content/suggested-content.njk]: Delete this one we don't use this file we use the macros.. it's a tech debt to get ride of all of them  😅

Comment (psoleckimoj) [reply moj-node/server/views/components/suggested-content/suggested-content.njk]: 😆

**Comment by psoleckimoj** [comment]
> Add a tests for the content routes so they expect `suggested content`

@romidane Not sure what to test here, `suggestedContentFor` is being tested elsewhere and it's not a route itself, might need to chat about this as it's a bit confusing, looking at what is an API endpoint in the config, and what is in the code. Would be nice for every endpoint to be in the config and used as such so mappings from what's in Drupal are easy to find/follow in the code. Some Tech Debt for us 😄

File: moj-node/test/routes/content.spec.js [hunk 6]
```diff
@@ -187,17 +204,29 @@ describe('GET /content/:id', () => {
...
             'Article stand first did not match',
           );
           expect($('#body').text()).to.include(
-            'Foo article body',
+            'Foo paragraph',
             'Article body did not match',
           );
+
+          // suggestions
+          expect($('#suggested-content a').length).to.equal(1);
```
Comment (romidane) [inline moj-node/test/routes/content.spec.js]: Nice.. I know this is tech debt but any chance you can break up this test..
As in one render and 3 smaller tests?

Comment (psoleckimoj) [reply moj-node/test/routes/content.spec.js]: @romidane done I believe!

**Pull request merged**